Src/BootCamp.Chapter/Person.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
 {
     public class Person
     {
-        // add missing properties
+        public string Name { get; set; }
+        public string BirthDay { get; set; }
+        public string SureName { get; set; }
+        public string Gender { get; set; }
+        public string Country { get; set; }
+        public string Email { get; set; }
+        public string StreetAdress { get; set; }
+
+        public int CalculateAge()
```

Comment:
Age should be a get-only property. You can ALWAYS easily get the age of a person based on birthday.

Reply:
thanks. changed it